src/pydap/parsers/__init__.py

```diff
@@ -110,19 +114,20 @@ def parse_ce(query_string, protocol="dap2"):
     """
     if protocol == "dap2":
         key = "&"
-        tokens = [token for token in unquote(query_string).split(key) if token]
-        if not tokens:
-            projection = []
-            selection = []
-        elif re.search("<=|>=|!=|=~|>|<|=", tokens[0]):
-            projection = []
-            selection = tokens
-        else:
-            projection = parse_projection(tokens[0])
-            selection = tokens[1:]
-    else:
+    elif protocol == "dap4":
+        key = "|"
+        # remove `dap4.ce=` from query string
+        query_string = query_string[8:]
```

Comment:
Does the code check that the 8 chars are actually _dap4.ce=_? It should, but if it's done elsewhere, that's good enough.